deploy_dod_dataset.py

```diff
@@ -49,39 +56,55 @@ def persist(self):
         return
 
 
-def deploy():
-    """The entry function for invokation
-
-    """
-    states_blob = {
-        'key': 'states.csv',
-        'body': get_usa_by_states_df().to_csv()
+def upload_csv(key_name, csv): 
+    blob = {
+        'key': f'{key_name}.csv',
+        'body': csv
         }
-    statesObj = DatasetDeployer(**states_blob)
-    statesObj.persist()
+    obj = DatasetDeployer(**blob)
+    obj.persist()
+    logger.info(f"Generated csv for {key_name}")
 
-    counties_blob = {
-        'key': 'counties.csv',
-        'body': get_usa_by_county_with_projection_df().to_csv()
-        }
-    countiesObj = DatasetDeployer(**counties_blob)
-    countiesObj.persist()
-
-    states_shp = BytesIO()
-    states_shx = BytesIO()
-    states_dbf = BytesIO()
-    get_usa_state_shapefile(states_shp, states_shx, states_dbf)
-    DatasetDeployer(key='states.shp', body=states_shp.getvalue()).persist()
-    DatasetDeployer(key='states.shx', body=states_shx.getvalue()).persist()
-    DatasetDeployer(key='states.dbf', body=states_dbf.getvalue()).persist()
-
-    counties_shp = BytesIO()
-    counties_shx = BytesIO()
-    counties_dbf = BytesIO()
-    get_usa_county_shapefile(counties_shp, counties_shx, counties_dbf)
-    DatasetDeployer(key='counties.shp', body=counties_shp.getvalue()).persist()
-    DatasetDeployer(key='counties.shx', body=counties_shx.getvalue()).persist()
-    DatasetDeployer(key='counties.dbf', body=counties_dbf.getvalue()).persist()
+def deploy(should_run_validation):
```

Comment:
I think this is a good case to use a keyword argument.  Generally with booleans I prefer it, since:
`deploy(True)` is a bit more confusing just skimming through than `deploy(should_run_validation=True)`